R/model-module-helpers.R

```diff
@@ -120,11 +122,11 @@ latest_obs_as_matrix <- function(latest) {
 convolution_matrix <- function(dist, t, include_partial = FALSE) {
   if (is.list(dist)) {
     if (length(dist) != t) {
-      stop("dist must equal t or be the same for all t (i.e. length 1)")
+      rlang::abort("dist must equal t or be the same for all t (i.e. length 1)")
```

Comment:
```suggestion
      rlang::abort("`length(dist)` must equal `t` or be the same for all t (i.e. length 1)")
```
Bit confused by the second note here - how is that checked?